rules/DowngradePhp81/Rector/MethodCall/DowngradeIsEnumRector.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DowngradePhp81\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Type\ObjectType;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Tests\DowngradePhp81\Rector\MethodCall\DowngradeIsEnumRector\DowngradeIsEnumRectorTest
+ */
+final class DowngradeIsEnumRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Downgrades isEnum() on class reflection', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run(ReflectionClass $reflectionClass)
+    {
+        return $reflectionClass->isEnum();
+    }
+}
+CODE_SAMPLE
+
+                ,
+                <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run(ReflectionClass $reflectionClass)
+    {
+        return false;
+    }
+}
+CODE_SAMPLE
+            ),
+        ]);
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isObjectType($node->var, new ObjectType('ReflectionClass'))) {
+            return null;
+        }
+
+        if (! $this->isName($node->name, 'isEnum')) {
+            return null;
+        }
+
+        return $this->nodeFactory->createFalse();
```

Comment:
I think method_exists is needed, same like other downgrade rule

https://github.com/rectorphp/rector-downgrade-php/blob/a6fc6e7cf8dcb8a837cfbd49bf95c02b5339b30e/rules/DowngradePhp74/Rector/MethodCall/DowngradeReflectionGetTypeRector.php#L54

Reply:
Sounds good, I'll update it 👍